### DISCUSSION TITLE
Welcome to OpenRefine Discussions!

### DISCUSSION THREAD

**Discussion by antoine2711** [antoine2711/OpenRefine#222] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by antoine2711** [comment]
@mathieulb: bienvenue! Comment tu trouves ça ? Pas mal, non ?

Ciao, Antoine

**Comment by mathieulb** [comment]
non
